ironfish/src/assets/assetsVerifier.ts

```diff
@@ -114,4 +119,32 @@ export class AssetsVerifier {
       return { status: 'unverified' }
     }
   }
+
+  getAssetData(assetId: Buffer | string): VerifiedAssetMetadata | undefined {
+    return this.verifiedAssets?.getAssetData(assetId)
+  }
+
+  verifyWithMetadata(
```

Comment:
I think we could combine this with the `verify` function and use the same one for all instances